submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/db/data/SubmissionRequestDocDataRepository.kt

```diff
@@ -137,34 +137,31 @@ class SubmissionRequestDocDataRepository(
     suspend fun updateIndex(
```

Comment:
I think we should rename the function to `increaseIndex` in this case since it's changing its behavior